shopinvader_product/schemas/__init__.py

```diff
@@ -0,0 +1,2 @@
+from . import category
```

Comment:
To ease import into others modules, you should import models here...

```python

from category import ShortProductCategory, ProductCategory
...

```

When we need to import these schemas into others addons the syntax will be simpler.

```python

from odoo.addons.shopinvader_product.schemas import ProductCategory

```